### DISCUSSION TITLE
Dice without background

### DISCUSSION THREAD

**Discussion by ouyuxuanbridge** [Project-MONAI/MONAI#3985] [Q&A] [answered]
When I am using this model, following (https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unetr_btcv_segmentation_3d.ipynb)

I don't want to include background, so I make this change: 
dice_metric = DiceMetric(include_background=True, reduction="mean", get_not_nans=False)

But then dice become 0?

Could you help me please?
Thanks!

**Comment by Nic-Ma** [comment]
Hi @Ouyuxuan623 ,

Thanks for your interest here.
If you don't want to include background, why you set `include_background=True`?

Thanks.

**Reply by giemmecci** [reply]
> Hi @Ouyuxuan623 ,
> 
> Thanks for your interest here. If you don't want to include background, why you set `include_background=True`?
> 
> Thanks.

Hi, @Nic-Ma,
I've found this thread looking for clarifications on various metrics, and I find the description of the "include_background" parameter given in the documentation counterintuitive, which could explain the confusion for the original poster.

From the documentation:

> include_background (bool) – whether to skip Dice computation on the first channel of the predicted output. Defaults to True.

To me, this means that if I set "True" I will skip the Dice computation on the first channel ("whether to skip" -> True -> will be skipped).

I think writing the documentation as:

> include_background (bool) – whether to ~~skip~~ **perform** Dice computation on the first channel of the predicted output. Defaults to True.

Or something similar, would probably clarify any doubts.

Thanks!

**Reply by KumoLiu** [reply, marked as answer]
Thanks for the advice, just create a [PR](https://github.com/Project-MONAI/MONAI/pull/6573) to fix it.
